src/index.js

```diff
@@ -0,0 +1,38 @@
+import Header from './components/Header';
+import Main from './components/Main';
+import Footer from './components/Footer';
+import './style.css';
+
+const page = document.querySelector('#content');
+
+const render = (currentPage) => {
+  const pageFragment = document.createDocumentFragment();
+  pageFragment.appendChild(Header());
+  pageFragment.appendChild(Main.choose(currentPage));
+  pageFragment.appendChild(Footer());
+  page.appendChild(pageFragment);
```

Comment:
You could also use [`append`](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append):

```js
  page.append(Header(), Main.choose(currentPage), Footer());
```